packages/ripple-binary-codec/test/signing-data-encoding.test.js

```diff
@@ -4,6 +4,10 @@ const {
   encodeForSigningClaim,
   encodeForMultisigning,
 } = require('../src')
+const { XrplDefinitions } = require('../src/enums/xrpl-definitions')
+
+// This changes the Payment TransactionType from being 0 to being 30 (aka 0x001F when encoded)
+const definitions = require('./fixtures/definitions-with-diff-payment.json')
```

Comment:
ditto here with not using the hard-to-parse fixtures - it'll be easier to understand if you just modify the normal definitions.json